static/src/javascripts/lib/fetch.spec.js

```diff
@@ -0,0 +1,122 @@
+// @flow
+
+import fetch from 'lib/fetch';
+import sinon from 'sinon';
+
+let xhr;
+let requests;
+
+describe('Fetch util', () => {
+    beforeEach(() => {
+        requests = [];
+        xhr = sinon.useFakeXMLHttpRequest();
```

Comment:
From what we understand, sinon clobbers the global xml request object and then we restore it to the original value in the afterEach block. @SiAdcock has suggested we might not even have to do this anymore as Jest sandboxes each test suite but I've kept it in for now.